TACC/protx-dashboard
# task/COOKS-348 Move community characteristics under county name

## Overview: ##
Move the community characteristics modal to under the county name when a county is selected

## Related Jira tickets: ##

*[COOKS-348](https://jira.tacc.utexas.edu/browse/COOKS-348)

## Summary of Changes: ##

## Testing Steps: ##
1. Go to cep.test and choose analytics from the drop down
2. Make sure the community characteristics modal looks the same and work on click

## UI Photos:
Doesn't show when statewide data displayed
![Screen Shot 2023-02-21 at 12 33 46 PM](https://user-images.githubusercontent.com/96220951/220430052-399d2c43-31d9-42cc-85da-9c5728119d61.png)

Shows when a county is selected:
![Screen Shot 2023-03-29 at 1 59 46 PM](https://user-images.githubusercontent.com/96220951/228640783-77559174-ba22-4b69-a12f-1b4762ad41de.png)

Modal renders correctly and looks the same.
![Screen Shot 2023-02-21 at 12 35 03 PM](https://user-images.githubusercontent.com/96220951/220430294-fc8aeb4f-4749-40fe-b97d-ecebdb1b2e40.png)



## Notes: ##
